quadtree.js

```diff
@@ -90,7 +162,7 @@ class QuadTree {
       found = [];
     }
     if (!this.boundary.intersects(range)) {
```

Comment:
Instead of modifying the Rectangle intersects you can swap the objects into
```js
if(range.intersects(this.boundary)) {
```

Reply:
nice!
Thank you.